test/HealthChecks.UI.Tests/Functional/UIApiRequestLimitingTests.cs

```diff
@@ -47,18 +57,26 @@ public async Task should_return_too_many_requests_status_code_when_exceding_conf
 
             using var server = new TestServer(webHostBuilder);
 
-            var requests = Enumerable.Range(1, maxActiveRequests)
-                .Select(n => server.CreateRequest($"/healthchecks-api").GetAsync());
+            // warmup
+            (await server.CreateRequest("/healthchecks-api").GetAsync().ConfigureAwait(false)).StatusCode.ShouldBe(HttpStatusCode.OK);
+
+            var requests = Enumerable.Range(1, maxActiveRequests + 2)

```

Comment:
What about changing this to:

`var requests = Enumerable.Range(1, maxActiveRequests + 2).AsParallel()`

Reply:
I got it! It's wrong endpoint!

Reply:
fixed in https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/1869